app/app.go

```diff
@@ -985,7 +908,6 @@ func New(
 		dexModule,
 		marketmapModule,
 		oracleModule,
-		auction.NewAppModule(appCodec, app.AuctionKeeper),
 		consensus.NewAppModule(appCodec, app.ConsensusParamsKeeper),
 		// always be last to make sure that it checks for all invariants and not only part of them
 		crisis.NewAppModule(&app.CrisisKeeper, skipGenesisInvariants, app.GetSubspace(crisistypes.ModuleName)),
```

Comment:
out of scope, but, is there a reason, why `stateverifier` is the last module. If not, lets change the order, set crisis last here

Reply:
Fixed